jenkins/internal-ci/centos-7.9.2009/pr/cortx_deployment/pyutils.groovy

```diff
@@ -167,7 +167,6 @@ pipeline {
                     popd
                     cp "${THIRD_PARTY_LOCATION}/THIRD_PARTY_RELEASE.INFO" "${DESTINATION_RELEASE_LOCATION}"
                     cp "${CORTX_ISO_LOCATION}/RELEASE.INFO" "${DESTINATION_RELEASE_LOCATION}"
-
```

Comment:
I see there is no change in this file. Can we skip this file.

Reply:
yes